pkg/apis/veleroplugin/v1/upload.go

```diff
@@ -75,6 +76,22 @@ type UploadStatus struct {
 	// If the DataManager couldn't process Upload for some reason it will be picked up by another
 	// node.
 	ProcessingNode string `json:"processingNode,omitempty"`
+
+	// RetryCount records the number of retry times for adding a failed Upload which failed due to
+	// network issue back to queue. Used for user tracking and debugging.
+	// +optional
+	RetryCount int32 `json:"retryCount,omitempty"`
+
+    // NextRetryTimestamp should be the timestamp that indicate the next retry for failed upload CR.
+    // Used to filter out the upload request which comes in before next retry time.
+	// +optional
+	// +nullable
+    NextRetryTimestamp *meta_v1.Time `json:"nextRetryTimestamp,omitempty"`
+
+	// CurrentBackOff records the backoff on retry for failed upload. Retry on upload should obey
+	// exponential backoff mechanism.
+	// +optional
+	CurrentBackOff int32 `json:"currentBackOff,omitempty"`
```

Comment:
Just `Backoff` should be enough since there are no `PreviousBackoff` and `NextBackoff`.